src/settings.ts

```diff
@@ -16,3 +16,9 @@ export function getStyleSheets(uri: Uri): string[] {
     .getConfiguration("css", uri)
     .get<string[]>("styleSheets", []);
 }
+
+export function getVaildOnSave(): Boolean {
+  return workspace
```

Comment:
Should be resource scopped like styleSheets

Reply:
Is vscode-html-css modified to css?